src/gui/properties/peerlistwidget.cpp

```diff
@@ -479,6 +486,8 @@ void PeerListWidget::updatePeer(const BitTorrent::Torrent *torrent, const BitTor
     const QString downloadingFilesDisplayValue = downloadingFiles.join(u';');
     setModelData(row, PeerListColumns::DOWNLOADING_PIECE, downloadingFilesDisplayValue, downloadingFilesDisplayValue, {}, downloadingFiles.join(u'\n'));
 
+    m_proxyModel->filterRow(row, (peer.isConnecting() && !m_showConnectingPeers));
```

Comment:
```suggestion
    m_proxyModel->filterRow(row, (!m_showConnectingPeers && peer.isConnecting()));
```

Reply:
It is cheaper to check a variable than to make a function call.

Reply:
> It is cheaper to check a variable than to make a function call.

Well, actually, it's not a variable, but `CachedSettingValue`.

Reply:
> It is cheaper to check a variable than to make a function call.

the 'variable' is not a simple variable.